docs/resource_limits.md

```diff
@@ -0,0 +1,159 @@
+# Contour / Envoy Resource Limits
+
+## Performance Testing Contour / Envoy
+
+- Cluster Specs
+  - Kubernetes
+    - Version: v1.12.6
+    - Nodes:
+      - 5 Worker Nodes
+        - 2 CPUs Per Node
+        - 8 GB RAM Per Node
+      - 10 GB Network
+  - Contour
+    - Single Instance
+      - 4 Instances of Envoy running in a Daemonset
+      - Each instance of Envoy is running with HostNetwork
+    - Cluster Network Bandwidth
+
+Having a good understanding of the available bandwidth is key when it comes to analyzing performance. It will give you a sense of how many requests per second you can expect to push through the network you are working with.
+
+Use iperf3 to figure out the bandwidth available between two of the kubernetes nodes. The following will deploy an iperf3 server on one node, and an iperf3 client on another node:
```

Comment:
Should there be a command here?

Reply:
Ahh good call, I stole this from some other notes on how I set up the perf testing. I'll fix up.